nautobot/dcim/tests/test_views.py

```diff
@@ -632,6 +633,7 @@ def setUpTestData(cls):
     # factories.
     @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
     def test_bulk_edit_objects_with_constrained_permission(self):
+        Device.objects.all().delete()
```

Comment:
Are this one and those below redundant? We already deleted all Devices in `setUpTestData`, and the line below this is deleting all Devices except those of a specific test model?